src/cloudsc_sycl/cloudsc/load_state.cpp

```diff
@@ -1,22 +1,13 @@
-/*
- * (C) Copyright 1988- ECMWF.
- *
- * This software is licensed under the terms of the Apache Licence Version 2.0
- * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
- * In applying this licence, ECMWF does not waive the privileges and immunities
- * granted to it by virtue of its status as an intergovernmental organisation
- * nor does it submit to any jurisdiction.
```

Comment:
Looks like you accidentally dropped the header in this copy-paste. This needs to go back, I think.